runtime/imports.go

```diff
@@ -84,16 +84,13 @@ func ext_malloc(context unsafe.Pointer, size int32) int32 {
 	mutex.Lock()
 	runtimeCtx := registry[*(*int)(instanceContext.Data())]
 	mutex.Unlock()
-	log.Debug("[ext_malloc]", "context", *(*int)(instanceContext.Data()))
```

Comment:
Why are these being removed?

Reply:
too many logs when I run the tests, can't see what's going on

Reply:
I don't feel that's justification for removing them. If there is an issue with how many logs are in tests we should discuss how to solve that.

Reply:
this isn't in the tests, this is in the actual code. there's also duplicate logs inside the allocator package

Reply:
Solving this would be solved by hiding logs or limiting verbosity during tests. If you are saying there are too many logs when running gossamer, this will be fixed with `--verbosity`

Reply:
how do i limit verbosity during tests? I usually run with `-v` since I want to see my logs, but these logs add way too much stuff to look through

Reply:
That is something we need to address by either a) passing loggers into services, allowing us to disable them or b) setting up a log handler at the start of tests (some combination of `testing.Main()` and `log.SetHandler()`).

My suggestion here would be to add these back in and open an issue regarding logging issues.

Reply:
okay, I would like to remove them still since there are duplicate logs in allocate.go (which I haven't deleted) https://github.com/ChainSafe/gossamer/blob/development/runtime/allocator/allocator.go#L115

I don't think we need 2 logs telling us the heap size :p

Reply:
There's the answer I was looking for :star: Will open an issue to clean up the runtime logs, could definitely be improved